### PR TITLE
Briefly mention local tables in metadata

### PR DESCRIPTION
**Pull request by onderkalaci**

### CODE REVIEW

File: develop/api_udf.rst [hunk 1]
```diff
@@ -263,6 +263,47 @@ reference table
...
+tables (see :ref:`ddl` and :ref:`citus_sql_reference`).
+
+If you drop the foreign keys, Citus will automatically remove such local tables from metadata,
+which eliminates such limitations on those tables.
+
+Arguments
+************************
+
+**table_name:** Name of the table on the coordinator which needs to be accessed from workers.
+
```
Comment (onurctirtir) [inline develop/api_udf.rst:290]: should we mention cascadeViaForeignKeys option ?

Comment (onderkalaci) [reply develop/api_udf.rst:290]: When a table is added via this UDF, even if fkey is dropped, it stays on the metadata.

What do you think should we mention regarding cascadeViaForeignKeys?

Comment (onderkalaci) [reply develop/api_udf.rst:290]: Ah, I forgot about `cascade_via_foreign_keys` parameter, now I got what you mean.

**Review by onurctirtir — APPROVED** [review]

**Comment by jonels-msft** [comment]
Thanks!

I made a few edits and it looks good.

**Pull request merged**